.github/workflows/release.yml

```
@@ -0,0 +1,147 @@
+name: Release
+
+"on":
+  workflow_dispatch:
```

YAMLlint (1.37.1)

[error] empty value in block mapping

(empty-values)